virttest/utils_misc.py

```diff
@@ -2192,8 +2192,9 @@ def check_model_list(pattern):
 
     x86_pattern_list = "x86\s+\[?([a-zA-Z0-9_-]+)\]?.*\n"
     ppc64_pattern_list = "PowerPC\s+\[?([a-zA-Z0-9_-]+\.?[0-9]?)\]?.*\n"
-
-    for pattern_list in [x86_pattern_list, ppc64_pattern_list]:
+    s390_pattern_list = "s390\s+\[?([a-zA-Z0-9_-]+)\]?.*\n"
+    
```

Comment:
You left a trailing whitespace on this line, as the Travis job caught it:

    /home/travis/build/avocado-framework/avocado-vt/virttest/utils_misc.py:2196:1: W293 blank line contains whitespace

Can you please fix it and force push your branch?